packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/beginProposal.ts

```diff
@@ -25,18 +25,18 @@ const afterExecute = () => async (
 ): Promise<{ proposalId: string } | undefined> => {
   const events = response.responses[0].tx.events
   if (!events) {
-    logger.error('No events found. Proposal ID could not be retrieved')
+    logger.error('No events found. Config Proposal ID could not be retrieved')
     return
   }
 
   try {
     const proposalId = events[0].wasm.proposal_id[0]
-    logger.success(`New config proposal created with Proposal ID: ${proposalId}`)
+    logger.success(`New config proposal created with Config Proposal ID: ${proposalId}`)
```

Comment:
There is also some new external domain knowledge introduced here (log references a 'Config Proposal') that's irrelevant to the internals of this command.

No suggestion, just thinking out loud.